msg_helpers.go

```diff
@@ -474,6 +474,8 @@ Option:
 		}
 		edns = append(edns, e)
 		off += int(optlen)
+
+	// Process any unrecognized option code as a local/experimental option.
```

Comment:
I don’t think this comment adds much.

Reply:
I consider it important to note because `EDNS_LOCAL` at a glance _looks_ similar to the other `EDNS0_*` structures, but is in fact a generic container that can contain anything, including options that have been standardized but are not yet handled by this library.